controls/datetimepicker/commands.md

```diff
@@ -35,9 +35,9 @@ xmlns:telerik="http://schemas.telerik.com/2022/xaml/maui"
 
 ## PopupSelector Commands
 
-Through the popup users can pick a date. The date value has to be confirmed or rejected with the **OK** or **Cancel** buttons that are located on the popup.
+Through the popup users can pick a date. The date value has to be confirmed or rejected with the `OK` or `Cancel` buttons that are located on the popup.
```

Comment:
This was correct before editing. Buttons and other UI elements are formatted in **bold**. See the [Formatting article](https://docs.telerik.com/style-guide/element-formatting) for details.